wiki/Tournaments/Official_support/en.md

```diff
@@ -75,95 +129,43 @@ Profile badge images must adhere to the following standards:
 - The badge must clearly display the logo, motif, or name of your tournament.
 - The badge must be cleanly designed and of reasonable visual quality.
 
```

Comment:
- The badge must be **172x80**px in dimension.

Reply:
this is [already a requirement](https://github.com/ppy/osu-wiki/blob/36b83eba78ebdd178904a680b8226839994dd4df/wiki/Tournaments/Official_support/en.md?plain=1#L128) (click on collapsed lines to expand them in the pr diff)